docs/sphinxext.py

```diff
@@ -56,6 +56,14 @@
         "mask_maps.py": 4,
         "makers.py": 5,
         "datasets.py": 6,
+        "models.py": 1,
+        "priors.py": 2,
+        "model_management.py": 10,
+        "fitting.py": 11,
+        "nested_sampling_Crab.py": 12,
+        "estimators.py": 13,
+        "catalog.py": 8,
+        "astro_dark_matter.py": 3,
```

Comment:
Why the duplication here? Re-ordering of the tutorials should come separately no?

Reply:
Not sure what you are refering to ?
The big re-ordering of tutorials with discussed in #5796 is another thing which is not implemented here.
So far this PR only adds the nested sampling in the API category.
Re-ordering will be done in a separate PR.

Reply:
Ahh now I get it
some tutorial are indeed repeated. A bad copy/paste maybe.
Fixed now